src/components/basic/form/FieldSet.md

```diff
@@ -4,31 +4,53 @@
 
 ```jsx
 import FieldSet from "./FieldSet";
```

Comment:
not blocking, but you shouldn't have to import the actual component you are documenting, in this case fiedlset, react-styleguidist just knows.